### ISSUE TITLE
Multiple Birdseye Views

### ISSUE THREAD

**Issue by DaCeige** [blakeblackshear/frigate#6531]
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to define multiple birdseye views to separate my cameras.  I would like an 'inside' and and 'outside' birdseye view with my cameras separated to each.  In theory, I would like a third that still has all the cameras on it.  

**Describe the solution you'd like**
I would like to be able to define multiple birdseye views per camera.  Example:
```
# Optional: birdseye configuration
# NOTE: Can (enabled, mode) be overridden at the camera level
birdseye:
  # Optional:  Specify which birdseye views this camera should be added to (default: all cameras)
  views:
    - inside cameras
    - all cameras
    - outside cameras

```
**Describe alternatives you've considered**
I consider setting up a separate frigate instance, but as I understand it, I would not be able to 'share' one coral interface between them.  Alternatively, using the newer code, I could potentially separate the detector to run 2 frigate instances with one coral, but that seems overly complicated.  
Alternatively, could create a matrix view using individual cameras on a HA dashboard.

**Additional context**
Basically looking for a way to create a matrix view that contains different cameras.  Outside cameras are more useful during the day, inside at night and when we're away.

**Comment by toxic0berliner** [comment]
Would love that too!

**Comment by psilantropy** [comment]
Came here to post this idea also. 
My use-case is similar. 6 cameras, 4 outdoor, 2 indoor. With indoor cameras only activated when we are not home.

I found having 6+ cameras in a birdseye view was too many cameras for my liking. I also want these views to all be continuous.

- birdseye1 = dog cam view
- birdseye2 = outdoor x4 view

**Comment by tomgit33** [comment]
+1  -> 4 Cams each group, selectable/sortable

**Comment by michaellammers** [comment]
+1

**Comment by nzkiwi68** [comment]
Another up vote up for multiple Birdseye views.

I have a use case for a large scale Frigate deployment to replace 4 x NVR's running ~120 cameras.
This use case will need 4 or more Frigate servers, but, it will need the ability to have views.

Show me on Frigate server1:

- entrance cameras
- checkout cameras
- yard cameras


Show me on Frigate server2:

- checkout cameras
- loading dock cameras
- building2 outdoor cameras
- building2 internal cameras


etc...


This feature request plus [Authorisation Roles #6614](https://github.com/blakeblackshear/frigate/issues/6614) are the two features that I believe will lift Frigate up to make large scale Frigate possible.


**Recommendation:**
Make this a part of "Frigate+".

Thanks for an amazing product!

**Comment by toxic0berliner** [comment]
If it ain't the 20 years old UI that moved you away and towards opnsense then I don't really get you 🤣
I do believe authorization roles can understandably be more of an enterprise feature and therefore paid, but several birdseye for me makes a lot of sense for a lot of small OpenSource free consumers given frigate works well on cheap hardware with cheap cameras it can easily grow to over 9 cams and 2 birdseye make sense. I can setup a frigate with 12 cams for less than $500 wich can make a lot of sense for individuals that don't want a subscription and would still need or at least badly want 2 or more birdseye...

**Comment by toxic0berliner** [comment]
Hmmm, maybe I missed something @NickM-27 but in your PR I read the groups are a per-device setting I don't see how this can close having multiple birdseye request... 
Maybe I made too many assumptions but I would have expected a way to define birdseyeN in the config as showing came,came and so on, so in the end I could have 3 stream urls birdseye1 birdseye2 and 3 for instance. In your PR I seem to see all is happening in the frontend, am I wrong?

**Comment by NickM-27** [comment]
to be clear it was not my PR but @hawkeye217

in general birdseye acted as a way to view multiple cameras and see activity that is occurring on each of those cameras. The camera groups are a shared global config, the only thing that is device specific is changing the layout itself. This should serve the vast majority of use cases requiring multiple split camera views and easily being able to switch between them. If there is some birdseye specific use case where the frigate frontend is not usable then I think a new feature request should be created as the PR linked solves the original request (`Basically looking for a way to create a matrix view that contains different cameras. Outside cameras are more useful during the day, inside at night and when we're away.`)

**Comment by nzkiwi68** [comment]
Just thinking...

Allow an optional  "TAG" field to each camera under birdseye.
Multiple birdseye views would be created, one for each tag name.

```
cameras:
  entrance:
    ui:
      order: 1
    birdseye:
      order: 1
	  tag:
	  - outdoor
```

Add multiple BirdEye views, views based on cameras with the tag of, say "carpark" or "outdoors".
With tags, then, one camera could even be present in multiple views.

```
cameras:
  entrance:
    ui:
      order: 1
    birdseye:
      order: 1
	  tag:
	  - outdoor
	  - carpark
```

**Comment by DPattee** [comment]
This feature would solve a problem I have when attempting to replace a dedicated NVR monitor system. 

I need to be able to specify multiple restreams where each stream contains a continuous ordered matrix of 4 specific cameras - so, for example, 4 "birdseye" streams to cover 16 cameras, so that:

1) one of our dedicated displays could cycle through them. (The display already has a feature to cycle through rtsp streams full stream... using 4x bullseye matrix streams would reduce CPU usage on that machine since it would only have 1 stream to decade at a time, vs always having to decode 4 streams every time). 

2) A higher priority display can be locked to a specific birdseye view of only critical cameras. 

The PR that closed this feature request doesn't seem to solve the issue, unless camera groups are somethign that can be restreamed but I couldn't make that work.

**Comment by mateuszdrab** [comment]
Shame this was never properly addressed, I too would like to see a birdseye stream with configurable cameras in it to ensure that legacy devices with low compute power can display a combined stream instead of having to render multiple streams locally.

Worked around by using native ffmpeg options in go2rtc which I then feed to destination bypassing Frigate.
https://github.com/AlexxIT/go2rtc/issues/354#issuecomment-4857806019
